contracts/common/MetaTransactionMsgSender.sol

```diff
@@ -25,4 +18,17 @@ abstract contract MetaTransactionMsgSender is DSMath {
       return payable(msg.sender);
     }
   }
+
+  function isMetatransaction() internal view returns (bool) {
+    uint256 index = msg.data.length;
+    if (msg.sender == address(this) && index >= 52) {
```

Comment:
I would not be opposed to introducing `uint256 constant METATX_MIN_LENGTH = 52` here for further clarity